packages/circuits/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "echo 'no-build'",
     "test": "npx ts-mocha test/**/*.test.ts --exit",
-    "format": "prettier --ignore-path .prettierignore --write \"**/*.+(js|ts|json)\"",
+    "format": "prettier --ignore-path ../../.prettierignore --write \"**/*.+(js|ts|json)\"",
```

Comment:
`build` command should be added too, but I can do it after merge anyway

Reply:
Yes, was wondering why it was not there. I added to the shell script, let me add it.